conda_forge_tick/migrators/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .core import *
+from .migration_yaml import *
+from .arch import *
```

Comment:
If we are going to use `import *` here, we should add `__all__` entries to the submodules to keep the namespace from exploding.

Reply:
Probably better to just be explicit here

Reply:
Sure, just not `*` with no `__all__`